shared/templates/static/bash/disable_ctrlaltdel_burstaction.sh

```diff
@@ -0,0 +1,5 @@
+# platform = Red Hat Enterprise Linux 7, multi_platform_fedora
+
+SYSTEMCONF='/etc/systemd/system.conf'
+
+sed -i 's/^#CtrlAltDelBurstAction=.*/CtrlAltDelBurstAction=none/' "$SYSTEMCONF"
```

Comment:
Would prefer that the replace_or_append function is used here.

Reply:
thanks, didn't know about that one.